training/ItemTableViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  ItemTableViewController.swift
+//  training
+//
+//  Created by Benedykt Ziobro on 07/11/2017.
+//  Copyright © 2017 Benedykt Ziobro. All rights reserved.
+//
+
+import UIKit
+import HandySwift
+
+class ItemTableViewController: UITableViewController {
+    
+    var items: [String] = Array(repeating:"", count:10)
+    var isTrigeringTask = false
+    
+    let celIdentifier = "ItemTableViewCell"
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.rowHeight = UITableViewAutomaticDimension
+        tableView.estimatedRowHeight = 44.0 
+        isTrigeringTask = true
+        trigereTask()
+        generateData()
+       
+    }
+    
+    override func didReceiveMemoryWarning() {
+        isTrigeringTask=false
```

Comment:
why? so if application receive memory warning you will stop generating data?

Reply:
I changed the way off stopping generating data when the app go to background. Now I use NotificationCenter for observing app state, and depend of it state I generate data and update data. I hope that is good solution of problem.